client/pkg/client/openapi/configuration.go

```diff
@@ -105,7 +105,7 @@ func NewConfiguration() *Configuration {
 		Debug:            false,
 		Servers:          ServerConfigurations{
 			{
-				URL: "https://api.relay.sh",
+				URL: "https://api.stage.relay-infra.net",
```

Comment:
Avoid checking this file in, please. At some point we'll have an easier method of generating/testing the latest changes (and perhaps versioning appropriately). But for now, generating against staging works fine, but I'd prefer to avoid broadcasting our staging URLs anywhere.